bucket/imagemagick.json

```diff
@@ -19,7 +22,7 @@
         "conjure.exe",
         "convert.exe",
         "dcraw.exe",
-        "ffmpeg.exe",
+        ["ffmpeg.exe", "iffmpeg"],
```

Comment:
That's fine for me, I just was curious.

A fourth option would be removing this alias completely